packages/modular-scripts/src/buildPackage/index.ts

```diff
@@ -23,7 +20,13 @@ import getRelativeLocation from '../utils/getRelativeLocation';
 
 const outputDirectory = 'dist';
 
-const rimraf = promisify(_rimraf);
+const IGNORED_FILES = [
+  '!package.json',
+  '!src',
```

Comment:
Why do we need to include both `src` and the 'dist-cjs', 'dist-es', 'dist-types' folders here? `makeBundle` returns a publicPackageJson that has 'dist-cjs', 'dist-es', 'dist-types' in the `files` array already.

Reply:
These are ignores rather than includes - these are files we know will already be included by the packlist (or by people who have put them in the `files` key of the package.json already) so we should ignore copying again to the target directory.